tiler_swift/run_handler.py

```diff
@@ -105,6 +107,47 @@ def save_results( self, results, output_path ):
     print(f"Results saved to {output_path}/{result_file_name}")
 
 
+  def gen_tiles ( self, results ):
+    self._tile_pairs = []
+    for idx, pairs in enumerate(results):
+      self._tile_pairs.append({})
+      for name, tile_loc_size in pairs.items():
+        x = tile_loc_size[0]
+        y = tile_loc_size[1]
+        w = tile_loc_size[2]
+        h = tile_loc_size[3]
+        tile = self._tensors[name][x:x+w, y:y+h]
+        self._tile_pairs[idx][name] = tile
+
+  def save_tiles( self, output_path ):
```

Comment:
It feels like we can save some memory usage if we gen and save the tiles (one by one) in the same function? (Comparing to storing the whole tile pairs in memory first, then process and save each)

Not a big deal tho

Reply:
We can, maybe we can update this if we have time